test/integration/targets/nxos_bfd_global/tests/common/sanity.yaml

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
I don't see the other directories and files under `targets/nxos_bfd_global/`.  Did you mean to omit them?

Reply:
Fixed.